### PR TITLE
build: add auto-approve to Node libraries

### PR DESCRIPTION
**Pull request by sofisl**

### CODE REVIEW

File: synthtool/gcp/templates/node_library/.github/CODEOWNERS [hunk 1]
```diff
@@ -11,3 +11,6 @@
 {%- else %}
 *     @googleapis/yoshi-nodejs
```
Comment (bcoe) [inline synthtool/gcp/templates/node_library/.github/CODEOWNERS:12]: Thinking out load, are we going to need to add our auto-approve user to each sub-team that owns repos, e.g., if a telemetry team owns logging repos, does the auto-approve user need to be part of it.

File: synthtool/gcp/templates/node_library/.github/auto-approve.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+rules:
+- author: "release-please[bot]"
+  title: "^chore: release"
+  changedFiles:
+  - "package\\.json$"
+  - "CHANGELOG\\.md$"
+  maxFiles: 3
+- author: "gcf-owl-bot[bot]"
```
Comment (bcoe) [inline synthtool/gcp/templates/node_library/.github/auto-approve.yml:8]: I think we should do the `.OwlBot.lock` config in a separate PR. I'm personally comfortable with automating merging these files. But @chingor13 had some concerns we should talk through as a group.

**Review by bcoe — CHANGES_REQUESTED** [review]
I'm excited to turn on auto-merging releases, I think it might be worth discussing our OwlBot lock file merging with @chingor13 and @SurferJeffAtGoogle.

File: synthtool/gcp/templates/node_library/.github/CODEOWNERS (hunk 1 above)
Comment (sofisl) [reply synthtool/gcp/templates/node_library/.github/CODEOWNERS:12]: Yes, we do, but this is only if they want auto-approve and they have subteams. I will add this to the README as well.

**Review by JustinBeckwith — CHANGES_REQUESTED** [review]
If we land this as a `feat:` it's going to trigger release PRs.  Let's relabel this as `chore:`?

**Review by bcoe — APPROVED** [review]

**Review by JustinBeckwith — APPROVED** [review]

**Pull request merged**